docs/04-guides/03-burn-nfts.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_label: "How to burn NFTs"
+---
+
+# How to burn NFTs
+
+## What is burning?
+Completely destroying a Token/NFT is called burning. Burning will destroy the whole token and close the corresponding accounts. This results in
+- The NFT being eliminated from the blockchain
+- while the transactions around it obviously remain in the blockchain history
+- giving you back some SOL that was reserved for account rent
+
+## How to burn...
+To burn NFTs different instructions can be run depending on which kind of NFTs you are using. You can not just burn it like a SPL token. This would not close all the token accounts required. Normal NFTs can be burned through the [`BurnNft` instruction](/programs/token-metadata/instructions#burn-a-nft), Editions Prints require a different instruction called [`BurnEditionNft` instruction](/programs/token-metadata/instructions#instructions#burn-an-edition-nft). There are multiple options for you how to burn NFTs as a user or programmatically as a developer:
+
+### ... as a user?
+* **Websites**: there are web based tools created by the community which allow you to connect your wallet and burn NFTs like [sol-incinerator.com](https://www.sol-incinerator.com/).
+* **Wallets**: Some wallets like Solflare allow you to directly burn a NFT in your wallet by choosing the NFT and clicking something like `close`
+* **[Metaboss](https://metaboss.rs/burn.html)**: Metaboss is a CLI tool primarily targeted at developers which has a `burn` function. You can either burn a single NFT or burn a [list of NFTs](https://metaboss.rs/burn.html#burn-all).
+Assumed that you have metaboss installed already you just have to run `metaboss burn one -k <OWNER_KEYPAIR> --account <MINT_ACCOUNT>` where `<OWNER_KEYPAIR>` is the path to your filesystem wallet and `<MINT_ACCOUNT>`the NFTs address.
```

Comment:
Nit: you might format the metaboss command as a separate block like

```bash
metaboss burn one -k <OWNER_KEYPAIR> --account <MINT_ACCOUNT>
```
to make it easier to read.

Reply:
Thanks, done!